docs/blog/unettest.md

```diff
@@ -0,0 +1,288 @@
+# UNETTEST
+
+![What is the answer, O Deep Thought](https://docs-unettest.s3.us-east-2.amazonaws.com/o_deep_thought.png)
+
+When I write software, there are many ways to ask a question. I could go to
```

Comment:
When writing blog posts for NYPR, we want to use "we" instead of "I". This is going to require some doing, but it's important that we keep a consistent voice across all of our blog posts.

Reply:
in the next section, ##The Monolith and the Microservice, you use "we" and "I" much better. I don't mind it there as much so try to incorporate that voice up here in the intro.

Reply:
If you need more explicit help on how to convert everything, lmk